Samples/excel-add-in-custom-functions-using-web-workers/src/taskpane/taskpane.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global console, document, Excel, Office */
+
+Office.onReady(() => {
+  document.getElementById("executeCFWithoutWebWorker").onclick = executeCFWithoutWebWorker;
+  document.getElementById("executeCFWithWebWorker").onclick = executeCFWithWebWorker;
```

Comment:
Let's rename all the functions and elements with the term "execute" to "run" instead.